src/connector/mysql/conversion.rs

```diff
@@ -105,6 +105,8 @@ impl<'a> From<ParameterizedValue<'a>> for MyValue {
                 dt.second() as u8,
                 dt.timestamp_subsec_micros(),
             ),
+            #[cfg(feature = "postgresql")]
+            ParameterizedValue::IpAddress(ip_addr) => MyValue::Bytes(ip_addr.to_string().into_bytes()),
```

Comment:
hmm...

?

Reply:
I know this is kind of needed if you have both features enabled. But maybe the enum variant should not be under a feature flag then...